### ISSUE TITLE
change instrument mode

### ISSUE THREAD

**Issue by prjemian** [APS-USAXS/ipython-usaxs#36]
be sure we have these useful macros (check SPEC macro names, of course):

* [x] use_mode_usaxs
* [x] use_mode_saxs
* [x] use_mode_waxs
* [ ] use_mode_radiography
* [ ] use_mode_imaging
* [x] useModeOpenBeamPath

**Comment by prjemian** [comment]
names:

useModeBlackFly     useModeImaging      useModeOpenBeamPath useModePinSAXS      useModeRadiography  useModeSAXS         useModeUSAXS        useModeWAXS

**Comment by prjemian** [comment]
also want a current status selector

**Comment by prjemian** [comment]
Are all of these reported via one PV?  That's what the `9dc_usaxs.ui` screen shows.

```
usaxs@usaxscontrol /share1/USAXS_data $ cainfo 9idcLAX:USAXS_Pin:USAXSSAXSMode
9idcLAX:USAXS_Pin:USAXSSAXSMode
    State:            connected
    Host:             ioc9idclax.xray.aps.anl.gov:5064
    Access:           read, write
    Native data type: DBF_DOUBLE
    Request type:     DBR_DOUBLE
    Element count:    1
usaxs@usaxscontrol /share1/USAXS_data $ caget 9idcLAX:USAXS_Pin:USAXSSAXSMode.RTYP
9idcLAX:USAXS_Pin:USAXSSAXSMode.RTYP ao
usaxs@usaxscontrol /share1/USAXS_data $ caget 9idcLAX:USAXS_Pin:USAXSSAXSMode
9idcLAX:USAXS_Pin:USAXSSAXSMode 2
```

This mode is **USAXS**

**Comment by prjemian** [comment]
Matt thinks this is important

**Comment by prjemian** [comment]
see https://subversion.xray.aps.anl.gov/spec/beamlines/USAXS/trunk/macros/local/

**Comment by prjemian** [comment]
or https://subversion.xray.aps.anl.gov/trac/spec/browser/beamlines/USAXS/trunk/macros

**Comment by prjemian** [comment]
specifically https://subversion.xray.aps.anl.gov/trac/spec/browser/beamlines/USAXS/trunk/macros/local/usaxs_commands.mac

**Comment by prjemian** [comment]
```
USAXSSAXSMODE = EpicsSignal("9idcLAX:USAXS_Pin:USAXSSAXSMode", name="USAXSSAXSMODE")
# values for USAXSSAXSMode:
# -1  dirty, prior move did not finish correctly
#  1  SAXS and USAXS out of beam
#  2  USAXS in beam
#  3  SAXS in beam
#  4  WAXS in the beam
#  5  Imaging in
#  6  Imaging tuning mode. 
```

**Comment by prjemian** [comment]
with #68 created, USAXS mode code can be checked off

**Comment by prjemian** [comment]
don't need X-rays to finish or test
also related to #73

**Comment by prjemian** [comment]
with #84 & #85 created, looks as if this issue can be closed - testing in #86
